src/encoder.rs

```diff
@@ -276,7 +355,7 @@ impl<W: Write> Writer<W> {
     /// This borrows the writer. This preserves it which allows manually
     /// appending additional chunks after the image data has been written
     pub fn stream_writer(&mut self) -> StreamWriter<W> {
-        self.stream_writer_with_size(DEFAULT_BUFFER_LENGTH)
+        self.stream_writer_with_size(Self::DEFAULT_BUFFER_LENGTH)
```

Comment:
What about stream writer, actually? Should it error or return a `StreamWriter` for an animated image chunk? Not that it's critical because currently it simply allows one to write multiple `IDAT` images anyways. :sweat_smile: